### PR TITLE
Improvement/return focus to enlarge image button on preview close

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
On Document capture confirmation screen, when image preview is enlarged and screen reader user clicks on “Close” toggle button, they return to the confirmation screen but there’s no element in focus. The user has to go through the top of the page to search for an element to focus on again. 

Additionally in latest SDK designs the "Enlarge image" toggle button is at the bottom of the preview frame.

# Solution
- Make sure that when the enlarged image preview is closed, the focus is returned to the "Enlarge image" toggle button 
- Move "Enlarge image" toggle to bottom of preview frame

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: src/components/Confirm/Previews.js [hunk 1]
```diff
@@ -124,7 +124,12 @@ const Previews = localised(
               }}
             />
           ) : (
-            <PageTitle title={title} smaller={true} className={style.title} />
+            <PageTitle
+              title={title}
+              smaller={true}
```
Comment (ethan605) [inline src/components/Confirm/Previews.js:129]: Should this be
```suggestion
              smaller
```
only

File: src/components/PageTitle/index.tsx [hunk 2]
```diff
@@ -12,22 +12,24 @@ type Props = {
...
   const isFullScreen = useSelector<RootState, boolean | undefined>(
     (state) => state.globals.isFullScreen
   )
   const containerRef = useRef<HTMLSpanElement>(null)
 
   useEffect(() => {
-    containerRef.current && containerRef.current.focus()
-  }, [title, subTitle])
+    containerRef.current && shouldAutoFocus && containerRef.current.focus()
+  }, [title, subTitle, shouldAutoFocus, isFullScreen])
```
Comment (ethan605) [inline src/components/PageTitle/index.tsx:32]: I'd love to have an extra comment here to explain why `isFullScreen` needs to be in the dependencies list

**Review by ethan605 — APPROVED** [review]

File: src/components/Confirm/Previews.js [hunk 3]
```diff
@@ -124,7 +124,12 @@ const Previews = localised(
...
+              className={style.title}
+              shouldAutoFocus={methodNamespace !== 'doc_confirmation'}
```
Comment (stefaniacardenas) [inline src/components/Confirm/Previews.js:131]: Does this mean that for documents the confirmation screen won't be announced?

**Review by stefaniacardenas — CHANGES_REQUESTED** [review]

File: src/components/PageTitle/index.tsx (hunk 2 above)
Comment (kopijunkie) [reply src/components/PageTitle/index.tsx:32]: Ah, that's my mistake. I was initially trying to use `isFullScreen` to determine if it should be auto focused on but that wasn't the right conditional to use. Will remove it from the dependencies list.

File: src/components/Confirm/Previews.js (hunk 3 above)
Comment (kopijunkie) [reply src/components/Confirm/Previews.js:131]: This only affects the focused element to not be the screen (page) title like for this case where on closing the preview we want the focus to start from the `Enlarge image` button. Title is always announced as the component's `aria-live` is set to "assertive"

**Review by stefaniacardenas — APPROVED** [review]

**Review by ethan605 — APPROVED** [review]

**Pull request merged**